OpenAI-DotNet/Assistants/AssistantTool.cs

```diff
@@ -1,12 +1,27 @@
 using System.Text.Json.Serialization;
 using OpenAI.Chat;
+using OpenAI.Extensions;
 
 namespace OpenAI.Assistants
 {
     public sealed class AssistantTool
```

Comment:
Can we use the existing Tool type?

Reply:
https://github.com/openai/openai-openapi/blob/master/openapi.yaml#L7785 not sure why but they have separate type for this one, so I'd better stick to separate type too

Reply:
Only thing that matters is if the fields are different I guess.

No matter, I'll validate later